src/content/finding.ts

```diff
@@ -72,6 +72,12 @@ class FindHighlight extends HTMLSpanElement {
 
 }
 
+// TODO: Conditionally import customElements-polyfill needed for the chrome browser:
+if (!customElements) {
+    // tslint:disable-next-line
+    require("@webcomponents/custom-elements");
+}
```

Comment:
```suggestion
```

I don't understand why this is needed on your machine. `customElements` is defined in Chrome. Tridactyl works fine on the options page on Chrome if I remove these lines. customElements have been in Chrome since version 54 - https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry#bcd:api.CustomElementRegistry - which was released in 2016.

Reply:
I just tried to remove the lines again, and build with `yarn run build:chrome` and run with `yarn run run:chrome`, and i still get the error when i open a new tab and go to google.
Here is what it looks like:
![Screenshot](https://user-images.githubusercontent.com/48760011/83418992-aeffd080-a424-11ea-9c4b-878e0dd7e81f.png)

My operating system is Ubuntu 18.04 and the chrome version is 83.

Reply:
Ah, I see. I hadn't spotted that Tridactyl was working (a little bit) on pages that weren't the options page. Odd that Chrome doesn't support customElements in web extensions!